_layouts/article.html

```diff
@@ -102,6 +102,11 @@
 		align-items: center;
 	}
 
+
+	 .next {
+    float: right;
+  }
```

Comment:
Use tab indentation only:

```suggestion
	.next {
		float: right;
	}
```